app/controllers/application_controller.rb

```diff
@@ -20,6 +20,10 @@ class ApplicationController < ActionController::Base
 
   before_action :set_timezone
 
+  # Allow users to impersonate other uses in a non-production environment.
+  # See the `pretender` gem.
+  impersonates :user
```

Comment:
Or is it enough to not route the impersonation in prod (ie. there is a guard in `routes.rb`) ?